fireplace-swing/src/main/java/com/github/bric3/fireplace/flamegraph/FlameGraphPainter.java

```diff
@@ -36,6 +36,7 @@
     public boolean paintHoveredFrameBorder = true;
     public int frameBorderWidth = 1;
     public Stroke frameBorderStroke = new BasicStroke(frameBorderWidth);
+    public Color frameBorderColor = Color.WHITE;
```

Comment:
```suggestion
    public Color frameBorderColor = Colors.panelForeground;
```

The `Colors` class can react to LaF changes. And updates its colors, which can be useful for IJ